packages/components/src/index.ts

```diff
@@ -13,7 +13,7 @@ export { default as Ribbon } from './ribbon';
 export { default as RootChild } from './root-child';
 export { default as ScreenReaderText } from './screen-reader-text';
 export { useScrollToTop } from './scroll-to-top/use-scroll-to-top';
-export { SiteThumbnail } from './site-thumbnail';
+export { SiteThumbnail, DEFAULT_THUMBNAIL_SIZE } from './site-thumbnail';
```

Comment:
Looks like this change can be removed

Reply:
Just caught this one. Do you mean to not expose the constant? Otherwise, it needs to be exported from this package, so I can reuse it.